exercises/tom_clayton/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,194 @@
+/**
+ * Import statements
+ */
+
+#include <iostream>
+#include <sstream>
+#include <fstream>
+#include <algorithm>
+#include <cctype>
+#include <string.h>
+#include <vector>
+#include <map>
+
+/** Returns a string filepath from user input
+ *  @return string filepath
+ */
+
+std::string getInputFn() {
+  std::cout << "Please enter the path of the file to be analysed" << std::endl;
+  std::string fin;
+  std::cin >> fin;
+  return fin;
+}
+
+/** Split up a string and return a vector of words with
+ *  len <=4, and converted to lower case
+ *  @param s, reference to string s
+ *  @param delim, character to use as delimiter
+ * 
+ *  @return vector of strings as split by chosen delimiter
+ */
+
+std::vector<std::string> splitByDelim(const std::string& s, const char delim) {
+  
+  std::vector<std::string> tokens;
+  std::string token;
+  std::istringstream tokenStream(s);
+  
+  while (std::getline(tokenStream, token, delim)) {
+    if(token.size() <= 4) {
+      continue;
+    }
+
+    std::transform(token.begin(), token.end(), token.begin(),
+    [](unsigned char c){ return std::tolower(c);});
+    
+    tokens.push_back(token);
+  }
+
+  return tokens;
+}
+
+/** Returns different integer values to classify an input char as
+ *  either non-punctuation, puntuation to be removed or to be
+ *  replaced with whitespace.
+ *  Specified punctuation to be removed: .,?'"!()
+ *  Specified punctuation to be replaced with whitespace: -
+ * 
+ *  @param c - Reference to the character in question
+ *  @return int 0 if non-punct, 1 if punct to remove, 2 if punct to
+ *  replace
```

Comment:
Now that you've merged the original specPunt function into depunctutate, I don't think you need the comment on lines 53-61 any more?